CPAC/pipeline/engine.py

```diff
@@ -230,26 +244,14 @@ def get(self, resource, pipe_idx=None, report_fetched=False,
                     if report_fetched:
                         return (None, None)
                     return None
-                lookup_message = ("\n[!] C-PAC says: None of the listed "
-                                  "resources are in the resource pool:\n"
-                                  f"{resource}\n")
-                verbose_logger.debug(lookup_message)
-                raise Exception(lookup_message)
+                raise Exception(info_msg)
```

Comment:
I think I left these as different exception types in a previous change, but while we're updating here, maybe make this a more-specific `LookupError` like https://github.com/FCP-INDI/C-PAC/blob/75bbc1bef9e42760ea4a83ec05d72b92df3379c2/CPAC/pipeline/engine.py#L254?